llvm/lib/DWARFLinker/LLVM/DependencyTracker.cpp

```diff
@@ -9,8 +9,9 @@
 #include "DependencyTracker.h"
 #include "llvm/Support/FormatVariadic.h"
 
-namespace llvm {
-namespace dwarflinker_parallel {
+using namespace llvm;
```

Comment:
why switch to using here?

Reply:
This style is recommended by guidelines for namespaces in source file : https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions.
Here is the example of using it in codebase - https://github.com/llvm/llvm-project/blob/780a5116ba68ec8c53b65008b3407479478b2d5e/llvm/lib/DebugInfo/GSYM/DwarfTransformer.cpp#L26

The patch changes namespace structure so we need to update this place anyway. So it is updated using the recommended form.